kitty/fonts.c

```diff
@@ -586,14 +598,25 @@ set_sprite(GPUCell *cell, sprite_index x, sprite_index y, sprite_index z) {
     cell->sprite_x = x; cell->sprite_y = y; cell->sprite_z = z;
 }
 
+// Gives a unique (arbitrary) id to a box glyph
 static inline glyph_index
 box_glyph_id(char_type ch) {
 START_ALLOW_CASE_RANGE
     switch(ch) {
         case 0x2500 ... 0x259f:
-            return ch - 0x2500;
-        case 0xe0b0 ... 0xe0d4:
-            return 0xa0 + ch - 0xe0b0;
+            return ch - 0x2500; // IDs from 0x00 to 0x9f
+        case 0xe0b0 ... 0xe0b4:
```

Comment:
Note: there was an error here: `0xe0d4` instead of `0xe0b4`

Reply:
It is deliberate, its the whole block of powerline glyph ids, incase anymore are added in the future.